pkg/model/estimator/local/ratio_process.go

```diff
@@ -27,8 +27,51 @@ import (
 	"k8s.io/klog/v2"
 
 	collector_metric "github.com/sustainable-computing-io/kepler/pkg/collector/metric"
+	"github.com/sustainable-computing-io/kepler/pkg/config"
 )
 
+// getTransferredDataFromMetric is used when there is Hardware counter so we have to use cgroup or eBPF (CPUTime)
```

Comment:
`getTransferredDataFromMetric is used when there is Hardware counter so we have to use cgroup or eBPF (CPUTime)`
->
`getTransferredDataFromMetric is used when there is no Hardware counter so we have to use cgroup or eBPF (CPUTime)`